lib/_stream_readable.js

```diff
@@ -846,7 +846,9 @@ Readable.prototype.removeListener = function(ev, fn) {
 };
 
 Readable.prototype.removeAllListeners = function(ev) {
-  const res = Stream.prototype.removeAllListeners.call(this, ev);
+  const res = arguments.length === 0 ?
```

Comment:
Actually, could you do this instead: `Stream.prototype.removeAllListener.apply(this, arguments)`?

Reply:
You're right.